### DISCUSSION TITLE
Anvil block numbers keep popping up

### DISCUSSION THREAD

**Discussion by Teoaks** [Cyfrin/foundry-full-course-cu#519] [General]
Hello everybody. I am currently on the lesson 8, section about the interaction with the Metamask and the browser. In this section the command required to start anvil is ``make anvil`` because we hade Makefile folder that are allowing us to make custom commands, as far as I understood. 

When I type ``make anvil`` it starts the anvil with all the necessary data, but at the same time it keeps showing block numbers and it's hash and also the block time. It just keeps going and going...Could someone explain to me what is happening here? 

![Screenshot 2023-08-02 151109](https://github.com/Cyfrin/foundry-full-course-f23/assets/111569878/b8c4046d-6e45-492d-b54a-fbca28ef6d59)
I screenshooted the terminal for you to be able to see it better.

**Comment by shikhar229169** [comment]
In the Makefile the below line is written:
anvil :; anvil -m 'test test test test test test test test test test test junk' --steps-tracing --block-time 1
And due to which it actually logs the blocks being mined every 1 second.
So instead of running make anvil, just run anvil. :)

**Reply by Teoaks** [reply]
Thank you for you quick response!
